src/components/common/EditableInput.js

```diff
@@ -123,7 +127,7 @@ export class EditableInput extends (PureComponent || Component) {
       <div style={ styles.wrap }>
         <input
           style={ styles.input }
-          ref="input"
+          ref={ input => this.textInput = input }
```

Comment:
Since you changed this here, you also have to update where it was originally called as this.refs.input: https://github.com/casesandberg/react-color/pull/325/files#diff-4584130a5bc9a8717f47d3c842478b49R15